### DISCUSSION TITLE
taro 3.6.10版本 dev:weapp 失败，但dev:build正常 请问一下大家有遇到这问题吗? 如何解决的

### DISCUSSION THREAD

**Discussion by 18338625417** [NervJS/taro#14266] [Q & A]
resolve 'react' in 'D:\taro-react-wxapp\node_modules\react-reconciler\cjs'
  Parsed request is a module
  using description file: D:\taro-react-wxapp\node_modules\react-reconciler\package.json (relative path: ./cjs)
    aliased with mapping 'react': 'cjs/react.production.min.js' to 'cjs/react.production.min.js'
      Parsed request is a module
      using description file: D:\taro-react-wxapp\node_modules\react-reconciler\package.json (relative path: ./cjs)
        Field 'browser' doesn't contain a valid alias configuration
        resolve as module
          D:\taro-react-wxapp\node_modules\react-reconciler\cjs\node_modules doesn't exist or is not a directory
          looking for modules in D:\taro-react-wxapp\node_modules\react-reconciler\node_modules
            D:\taro-react-wxapp\node_modules\react-reconciler\node_modules\cjs doesn't exist
          D:\taro-react-wxapp\node_modules\node_modules doesn't exist or is not a directory
          looking for modules in D:\taro-react-wxapp\node_modules
            D:\taro-react-wxapp\node_modules\cjs doesn't exist
          D:\node_modules doesn't exist or is not a directory

→ Watching... [2023/7/31 10:02:33]

node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "[object Array]".] {
  code: 'ERR_UNHANDLED_REJECTION'
}

**Comment by HongxuanG** [comment]
升级3.6.11版本

**Reply by 18338625417** [reply]
已解决 官方这cli更新的太快了

**Reply by github-zhang** [reply]
如3.6.10 更新为 3.6.11 ，需要将nodejs更新为18，同时package.json 3.6.10 全部替换为 3.6.11
